crates/e2e/src/client.rs

```diff
@@ -31,6 +31,7 @@ use ink_env::Environment;
 use ink_primitives::MessageResult;
 use pallet_contracts_primitives::ExecReturnValue;
 use sp_core::Pair;
+#[cfg(feature = "std")]
```

Comment:
You don't really need this since client is always compiled with `std` feature

Reply:
a leftover - removed